README.md

```diff
@@ -10,15 +10,43 @@
 ![PyPI - Downloads](https://img.shields.io/pypi/dm/u8darts)
 ![GitHub Release Date](https://img.shields.io/github/release-date/unit8co/darts)
 
-**darts** is a python library for easy manipulation and forecasting time series.
+**darts** is a python library for easy manipulation and forecasting of time series.
 It contains a variety of models, from classics such as ARIMA to neural networks.
 The models can all be used in the same way, using `fit()` and `predict()` functions,
 similar to scikit-learn. The library also makes it easy to backtest models,
 and combine the predictions of several models and external regressors.
 
+## Table of Content
```

Comment:
Pls also add a link to the docs https://unit8co.github.io/darts/. It seems that github just changed UI and this URL is not that easy to find anymore.

Reply:
this link is in the documentation section, you suggest to add it to the Table of Content directly ?